packages/common/core-interfaces/Removing-IFluidRouter.md

```diff
@@ -95,15 +95,21 @@ const entryPoint = await container.getEntryPoint();
 ## Status
 
 <!-- prettier-ignore-start -->
-| API                                                                                  | Deprecated in        | Removed in           |
-| ------------------------------------------------------------------------------------ | -------------------- | -------------------- |
-| `IContainer.request` (except calling with "/")                                       | 2.0.0-internal.6.0.0 |                      |
-| `IDataStore.request` (except calling with "/")                                       | 2.0.0-internal.6.0.0 |                      |
-| `IContainer.IFluidRouter`                                                            | 2.0.0-internal.6.0.0 |                      |
-| `IDataStore.IFluidRouter`                                                            | 2.0.0-internal.6.0.0 |                      |
-| `request` and `IFluidRouter` on `ILoader` and `Loader`                               | 2.0.0-internal.6.0.0 |                      |
-| `request` and `IFluidRouter` on `IRuntime` and `ContainerRuntime`                    | 2.0.0-internal.6.0.0 |                      |
-| `request` and `IFluidRouter` on `IFluidDataStoreRuntime` and `FluidDataStoreRuntime` | 2.0.0-internal.6.0.0 |                      |
-| `request` and `IFluidRouter` on `IFluidDataStoreChannel`                             | 2.0.0-internal.6.0.0 |                      |
-| `getRootDataStore` on `IContainerRuntime` and `ContainerRuntime`                     | 2.0.0-internal.6.0.0 |                      |
+| API                                                                                          | Deprecated in        | Removed in           |
+| -------------------------------------------------------------------------------------------- | -------------------- | -------------------- |
+| `IContainer.request` (except calling with "/")                                               | 2.0.0-internal.6.0.0 |                      |
```

Comment:
Remove exception

Reply:
There's a different line noting this in favor of being more verbose